dist/provenance/MixedStorageProvenanceGraphManager.d.ts

```diff
@@ -4,11 +4,14 @@
 import { ProvenanceGraph } from './ProvenanceGraph';
 import { IProvenanceGraphManager } from './provenance';
 import { IProvenanceGraphDataDescription } from './ICmd';
+import { ILocalStorageProvenanceGraphManagerOptions } from './LocalStorageProvenanceGraphManager';
+import { IRemoteStorageProvenanceGraphManagerOptions } from './RemoteStorageProvenanceGraphManager';
 import { GraphBase } from '../graph/GraphBase';
+export declare type IMixedStorageProvenanceGraphManagerOptions = ILocalStorageProvenanceGraphManagerOptions | IRemoteStorageProvenanceGraphManagerOptions;
```

Comment:
@puehringer Should it not be an `&` since it is a combination of both?

```suggestion
export declare type IMixedStorageProvenanceGraphManagerOptions = ILocalStorageProvenanceGraphManagerOptions & IRemoteStorageProvenanceGraphManagerOptions;
```

Reply:
That's correct, I will change it.